### PR TITLE
test for direct accessing of faq article with url

### LINKED ISSUE #399: Add test verifying article link can be opened directly with query params
Add cypress test that will verify that if I open URL with `help` query string, it opens the specific article, e.g. `?help=%2Ffaq%2FRkFRQ2F0ZWdvcnk6OTA%3D%2Farticle%2FRkFRQXJ0aWNsZToxMjc%3D` will open "Trains and buses" article

Please note you need to slightly update index file to support query params for standalone SmartFAQ

### PR DESCRIPTION
**Pull request by dinodsaurus**
closes #399 <br/><br/><br/><url>LiveURL: https://smartfaq-cypress-query-params.surge.sh</url>

### CODE REVIEW

File: cypress/integration/QuerryParamsRedirects.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('Querry params redirect to corect article', () => {
+  it(`User should see article about cabin baggage`, () => {
+    cy.visit(
+      '/?cypress-query=/faq/RkFRQ2F0ZWdvcnk6ODk=/article/RkFRQXJ0aWNsZTo0MQ==',
+    );
+    cy.get('[data-cy=faq-breadcrumbs]').as('faq-breadcrumbs');
+    cy.get('[data-cy=faq-article-form]').as('faq-article-form');
+
+    cy
```
Comment (jaroslav-kubicek) [inline cypress/integration/QuerryParamsRedirects.js]: I would suggest to use `it('...')` to describe that 1) there are breadcrumbs 2) form 3) article with such title

Comment (dinodsaurus) [reply cypress/integration/QuerryParamsRedirects.js]: thanks for the suggestion! 😊

**Comment by git-toni** [comment]
This is great. Just to *very* minor things :slightly_smiling_face: 
- Why call the query key `cypress-query` and not `help` as in frontend? That would be easier for us to remember when testing manually our bundle.
- `Querry` -> `Query` and `Querys` -> `Queries` :smile:

**Comment by dinodsaurus** [comment]
Thanks @git-toni didnt know about the `help` part on the frontend

File: .circleci/config.yml [hunk 2]
```diff
@@ -77,6 +77,11 @@ jobs:
           command: |
             wget --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 -t 20 http://localhost:8080
             :
+      - run:
```
Comment (dinodsaurus) [inline .circleci/config.yml]: this fixes when cypress first 4-5 cypress tests fail because `webpack-server` is not running properly.

Comment (git-toni) [reply .circleci/config.yml]: Just wondering, should this `sleep` be before the `wget` test tho?

Comment (dinodsaurus) [reply .circleci/config.yml]: nope, `wget` tests if server responds, and as you can notice on your local machine `webpack-server` starts responding but its still not ready to be used, and then this sleep fixes that issue 🙂

**Pull request merged**